File: backend/src/main/java/com/team05/petmeeting/domain/user/dto/emailverify/EmailVerifyReq.java
package com.team05.petmeeting.domain.user.dto.emailverify;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Pattern;

public record EmailVerifyReq(
        @Email
        @NotBlank
        String email,

        @Pattern(regexp = "\\d{6}", message = "인증 코드는 6자리 숫자여야 합니다.")
        String code
) {
}

File: backend/src/main/java/com/team05/petmeeting/domain/user/dto/emailverify/EmailVerifyReq.kt
package com.team05.petmeeting.domain.user.dto.emailverify

import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.Pattern

data class EmailVerifyReq(
    @field:Email
    @field:NotBlank
    val email: String,

    @field:Pattern(regexp = "\\d{6}", message = "인증 코드는 6자리 숫자여야 합니다.")
    @field:NotBlank
    val code: String
)
